.github/workflows/pronto.yml

```diff
@@ -0,0 +1,40 @@
+name: Pronto
```

Comment:
Copied this file from `Rainforest` main repo. I just had to make some minor adjustments because the main app defines a specific `pronto` gems group which we don't have here.